packages/nc-gui/components/cell/Url.vue

```diff
@@ -47,28 +47,30 @@ const isExpandedFormOpen = inject(IsExpandedFormOpenInj, ref(false))!
 
 const isForm = inject(IsFormInj)!
 
+const trimVal = (val:string) => val && (val + '').trim();
```

Comment:
if the type is `string`, then we don't need to check `val` and add `' '` (?)

Reply:
it's still possible to have non-string value here so as an extra precaution I kept it.